packages/wallet/src/persistence/pouchdbStores/PouchdbStore.ts

```diff
@@ -0,0 +1,35 @@
+import { Logger } from 'ts-log';
+import { toPouchdbDoc } from './util';
+import PouchDB from 'pouchdb';
+
+export abstract class PouchdbStore<T> {
+  protected readonly logger: Logger;
+  protected readonly db: PouchDB.Database<T>;
+
+  constructor(public dbName: string, logger: Logger) {
+    this.logger = logger;
+    this.db = new PouchDB<T>(dbName);
+  }
+
+  /**
+   * Only used internally and for cleaning up after tests.
+   * If you need to use this for other purposes, consider adding clear() or destroy() to stores interfaces.
+   */
+  async clearDB(): Promise<void> {
```

Comment:
Won't it be needed in some form to cleaning up when wallet is being removed? Now it isn't just a matter of clearing KeyAgent, but also clearing storage.

Reply:
Good point. Wallets would want to 'destroy' the storage. This `clearDB` function keeps the database, but removes all docs from it. I'll add `destroy()` to the storage interfaces.

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/200/commits/8a1fc09dcdfc42258fd128eabea76a4ac5c5946f